axelor-supplychain/src/main/java/com/axelor/apps/supplychain/web/AnalyticDistributionLineController.java

```diff
@@ -0,0 +1,25 @@
+package com.axelor.apps.supplychain.web;
+
+import com.axelor.apps.account.db.InvoiceLine;
+import com.axelor.exception.AxelorException;
+import com.axelor.exception.service.TraceBackService;
+import com.axelor.rpc.ActionRequest;
+import com.axelor.rpc.ActionResponse;
+
+public class AnalyticDistributionLineController {
+
+  public void linkWithOrderLine(ActionRequest request, ActionResponse response)
+      throws AxelorException {
```

Comment:
please remove this throws declaration as the controller does not throw AxelorException